packages/serverless-contracts/src/contracts/apiGateway/utils/combineUrls.ts

```diff
@@ -0,0 +1,10 @@
+/// <reference lib="dom" />
```

Comment:
Is it really necessary to add this?
I am afraid that it would define global dom types for all the users of the package, even if they use the package only in the backend.

If it is mandatory, we'll have to think of a workaround.

Reply:
This is usefull when using fetch, it add type def without any package
In this case, I don't think it is useful so you can remove it